### ISSUE TITLE
Allow for two finger touch to rotate view in editor viewport

### ISSUE THREAD

**Issue by blackears** [godotengine/godot-proposals#8922]
### Describe the project you are working on

General Level Editing

### Describe the problem or limitation you are having in your project

IT is currently not easy to rotate the viewport camera using the touchpad when working on a laptop.  You can currently zoom using just the touchpad - it would be useful to add rotation to this too.

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

Other software programs let you rotate the view by dragging two fingers on the touchpad in parallel and zoom by pinching them together or pulling them apart.  It would be useful to have the viewport react in the same way.

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

The part of the editor viewport which currently implements pinch zooming should also handle rotation.

### If this enhancement will not be used often, can it be worked around with a few lines of script?

No

### Is there a reason why this should be core and not an add-on in the asset library?

Requires changing editor.

**Comment by AThousandShips** [comment]
Duplicate of https://github.com/godotengine/godot-proposals/issues/2295
